src/NzbDrone.Core/Download/Clients/DownloadStation/Proxies/DownloadStationProxy.cs

```diff
@@ -39,12 +39,12 @@ public void AddTorrentFromData(byte[] torrentData, string filename, string downl
                 arguments.Add("destination", downloadDirectory);
             }
 
-            arguments.Add("file", new Dictionary<string, object>() { { "name", filename }, { "data", torrentData } });
+            arguments.Add("file", new Dictionary<string, object>() { { "name", filename }, { "data", data } });
            
             var response = ProcessRequest(DiskStationApi.DownloadStationTask, arguments, settings, $"add torrent from data {filename}", HttpMethod.POST);            
```

Comment:
"add torrent" is no longer accurate. Same for other locations. replace it with 'task' and it's ok.